stdlib/Profile/src/Profile.jl

```diff
@@ -503,13 +503,22 @@ function flatten(data::Vector, lidict::LineInfoDict)
     return (newdata, newdict)
 end
 
+const SRC_DIR = normpath(joinpath(Sys.BUILD_ROOT_PATH, "src"))
+const LIB_DIR = normpath(joinpath(Sys.BUILD_ROOT_PATH, "usr", "lib"))
```

Comment:
When would this ever get baked into the compiled libraries for Profile? it is also commonly wrong since many distros will move this folder. It seems like you actually want `abspath(Sys.BINDIR, Base.LIBDIR)` here, at runtime?

Reply:
Makes sense. I was only testing with a local bare `make`